interop/src/main.rs

```diff
@@ -65,9 +65,9 @@ impl From<&Alpn> for Vec<Vec<u8>> {
     fn from(alpn: &Alpn) -> Vec<Vec<u8>> {
         match alpn {
             Alpn::H3 => vec![quinn_h3::ALPN.into()],
-            Alpn::Hq => vec![b"hq-29"[..].into()],
+            Alpn::Hq => vec![b"hq-31"[..].into()],
```

Comment:
I wonder, should we actually leave this at 29 for a while? From the mailing list it looks like large deployments are sticking to the draft-29 version identifiers even as they deploy improvements from 30 and later.

Reply:
Done.